doc_src/cmds/history.rst

```diff
@@ -11,7 +11,7 @@ Synopsis
 	\	\   [ *SEARCH_STRING...* ]
 | ``history`` **delete** [**--case-sensitive**]
 	\	\   [**--exact** | **--prefix** | **--contains**] *SEARCH_STRING...*
-| ``history`` { **merge** | **save** | **clear** | **clear_session** }
+| ``history`` ( **merge** | **save** | **clear** | **clear-session** )
```

Comment:
I'd drop the braces/parentheses here entirely.

Reply:
If you think they really need to be clearly delineated as mutually-exclusive, I'd put them on separate lines.

Reply:
yeah I think separate lines work well here